packages/vuetify/src/composables/form.ts

```diff
@@ -56,22 +63,33 @@ export const makeFormProps = propsFactory({
     type: Boolean as PropType<boolean | null>,
     default: null,
   },
+  isReactiveErrors: {
+    type: Boolean,
+    default: false,
+  },
 })
 
 export function createForm (props: FormProps) {
   const model = useProxiedModel(props, 'modelValue')
 
   const isDisabled = computed(() => props.disabled)
   const isReadonly = computed(() => props.readonly)
+  const isReactiveErrors = computed(() => props.isReactiveErrors)
```

Comment:
This computed property doesn't need to exist

Reply:
i fixed it